pkg/controller/cincinnati/new.go

```diff
@@ -225,18 +225,24 @@ func (k *kubeResources) newPolicyEngineRoute(instance *cv1beta1.Cincinnati) *rou
 }
 
 func (k *kubeResources) newEnvConfig(instance *cv1beta1.Cincinnati) *corev1.ConfigMap {
+	// If IPv6 boolean property is not configured in the Spec
```

Comment:
I added Address to the spec because the spec is used for rendering the template for one of the ConfigMaps, I didn't know if there is a better way to do that, but in this case, we only use that field for rendering the CM, I don't see any value on persisting its value in the api-server. Maybe there is a better solution?

Reply:
Binding to "::" works on IPv4 and IPv6 clusters. I reverted the changes from the API types.